tools/install-tcmalloc.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+#
+# Script to install libtcmalloc.so in the current WiredTiger git
+# repository.
+#
+# This is serves the same purpose, but is NOT THE SAME as:
+#
+# test/evergreen/tcmalloc_install_or_build.sh
+#
+# and the tcmalloc source version referenced here should be updated in
+# sync with that script.
+
+set -euf -o pipefail
+
+die() { echo "$@"; exit 1; }
+
+HERE=$(git rev-parse --show-toplevel) || die "FATAL Run in root of git workspace"
+[[ -f ${HERE}/CMakeLists.txt ]] || die "Does not look like a WT workspace - no CMakeLists.txt"
+[[ -d ${HERE}/TCMALLOC_LIB ]] && die "TCMALLOC_LIB already present"
+
+# Before building look for a prebuilt copy cached on this system.
+SRC_ID=mongo-SERVER-85737
+CACHED_DIR=${HOME}/.cache/wiredtiger/tcmalloc/${SRC_ID}
+CACHED_BIN=${HOME}/.cache/wiredtiger/tcmalloc/${SRC_ID}/TCMALLOC_LIB.tgz
+if [[ -r ${CACHED_BIN} ]]; then
+    echo "Installing cached copy from $CACHED_BIN"
+    tar zxf $CACHED_BIN
+    exit
+fi
```

Comment:
I try to think about scenarios where this script needs to be run:

1. When a new workstation is spawned from Evergreen. As no prebuilt copy is in place, this code section will be a "no-op" when running this script. 
2. When a newer tcmalloc version becomes available to which we hope to upgrade. There's a prebuilt copy in place while outdated. Running this script would extract the tarball with a dated version and exit (on line #28). 

Do we expect users to be aware of the version upgrade and edit line #22 for the 2nd scenario?

Reply:
This functionality has been removed.